frontend/ui/elements/common_info_menu_table.lua

```diff
@@ -7,6 +7,11 @@ local T = require("ffi/util").template
 
 local common_info = {}
 
+
+if Device:isDesktop() then
+    common_info.font_settings = require ("ui/elements/font_settings"):getMenuTable()
```

Comment:
Putting this in the *common* info menu and then disabling it in the file manager with the MenuSorter is pretty weird. ;-)

Reply:
I created it because there was a lot of duplication that was annoying to deal with.

Reader-specific stuff should simply go around/after here:

https://github.com/koreader/koreader/blob/33e4c36a422d8665fd2e330f930f94446dfe4529/frontend/apps/reader/modules/readermenu.lua#L144-L152

Reply:
Wow, I just requested your review, you're fast.

Sorry, I'm not familiar with the menu sorter. Where should I put the menu?, somewhere inside the reading app?.

Reply:
Just a coincidence, I saw your latest commit before you did that.